keyboards/telophase/telophase.h

```diff
@@ -62,6 +62,7 @@
 	{ k36, k37, k38, k39, k40, k41,      k42, k43, k44, k45, k46, k47 }, \
 }
 
+#define FLIP_HALF
```

Comment:
Is there a reason that this is added here?

Reply:
I see, it doesn't do anything over here. I was under the impression this was required. Thanks!